apps/mull-ui/src/app/components/event-page-info/event-page-info.spec.tsx

```diff
@@ -1,17 +1,35 @@
+import { MockedProvider } from '@apollo/client/testing';
```

Comment:
Same comments as for `apps/mull-ui/src/app/components/event-card/event-card.spec.tsx`

Reply:
Addressed